src/latexify/codegen/expression_codegen.py

```diff
@@ -240,6 +240,129 @@ def _generate_transpose(self, node: ast.Call) -> str | None:
         else:
             return None
 
+    def _generate_determinant(self, node: ast.Call) -> str | None:
+        """Generates LaTeX for numpy.linalg.det.
+        Args:
+            node: ast.Call node containing the appropriate method invocation.
+        Returns:
+            Generated LaTeX, or None if the node has unsupported syntax.
+        Raises:
+            LatexifyError: Unsupported argument type given.
+        """
+        name = ast_utils.extract_function_name_or_none(node)
+        assert name == "det"
+
+        if len(node.args) != 1:
+            return None
+
+        func_arg = node.args[0]
+        if isinstance(func_arg, ast.Name):
+            return rf"\det \left( \mathbf{{{func_arg.id}}} \right)"
```

Comment:
It looks some generation rules don't follow the convention of the library: `\mathopen{}\left*` and `\mathclose{}\right*`